package.json

```diff
@@ -22,14 +22,18 @@
     "chroma-js"
   ],
   "sideEffects": false,
-  "main": "./dist/index.js",
-  "unpkg": "./dist/index.umd.js",
-  "module": "./dist/index.module.js",
-  "source": "./src/index.ts",
+  "main": "./dist/index.main.umd.js",
```

Comment:
why is the "main" a UMD? you could do that in a "browser" perhaps, but node doesn't need a UMD (and really, neither does anyone else)

Reply:
The UMD is primarily for raw browser usage for both unpkg and jsdeliver. I think skypack does a better job at kind of use case but I liked the idea of a legacy catch-all UMD bundle.

unpkg:

> If you omit the file path (i.e. use a “bare” URL), unpkg will serve the file specified by the `unpkg` field in package.json, or fall back to `main`.
>
> [[source]](https://unpkg.com/)

jsdelivr:

> For packages hosted on npm, we support setting a "default" file for each package. This file is displayed at the top of directory listings on our website and available under shorter URL on the CDN. The default file can be configured by setting one of the following fields in package.json (ordered by priority):
>
> 1. jsdelivr
> 2. browser
> 3. main
>
> [[source]](https://www.jsdelivr.com/features#publishing-packages)

Reply:
Right - I think it's best not to penalize node users, or modern bundler users, by giving them a UMD. The "unpkg" key seems fine to include a UMD.